scripts/jenkins-release.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+[ -d venv ] || virtualenv venv
+pip install -e .
+
+if [ "$ENVIRONMENT" = "Select one" -o "$ENVIRONMENT" = "" ]; then
+  >&2 echo "You must select an environment."
+  exit 1
+fi
+
+if [ "$APPLICATION_NAME" = "Select one" -o "$ENVIRONMENT" = ""]; then
+  >&2 echo "You must select an application name."
+  exit 1
+fi
+
+if [ "$ENVIRONMENT" = "preview" ]; then
```

Comment:
What do you think about having one AWS_PROFILE per stage, instead of one per AWS account (i.e. preview/staging/production profiles)? That way we could either use the same access keys for staging/production and get the same behaviour or look into separating permissions where possible